osquery/events/tests/producer.cpp

```diff
@@ -8,7 +8,10 @@
  *
```

Comment:
I would recommend naming these producer/consumer tests something related to our kernel buffering/eventing implementation: `osquery/events/tests/kernel_{producer,consumer}.cpp`.